### DISCUSSION TITLE
搜图功能发不出链接

### DISCUSSION THREAD

**Discussion by xiaorunrun520** [Tsuk1ko/CQ-picfinder-robot#380] [问题求助]
就挺神奇的，日志里显示发出去了，也没报错什么的，但对面就是看不到
![image](https://user-images.githubusercontent.com/115028124/207373098-a159abac-f615-4f24-9ee8-fd42390ff1be.png)
![image](https://user-images.githubusercontent.com/115028124/207373153-67af8ed5-5146-4a2b-9927-007f7b44bf13.png)

**Comment by Tsuk1ko** [comment]
可能风控，把机器人挂几天再试试

**Reply by xiaorunrun520** [reply]
一段时间没开，现在又重新打开了，但是它发出来这个东西
![image](https://user-images.githubusercontent.com/115028124/215419873-89f6c51c-fae4-4b38-843f-5a7226f532e6.png)
然后npm的运行日志
![image](https://user-images.githubusercontent.com/115028124/215419978-07e73d25-2c20-4fb7-8c5c-3248fbb9fecb.png)

**Reply by xiaorunrun520** [reply]
又试了一下发现anime和doujin功能可以发出来，其他都发不出来

**Reply by xiaorunrun520** [reply]
破案了，被和谐了，把隐藏缩略图的配置打开了就发的出来了
但pixiv和链接是红的不知道会不会被封
但是搜本子搜动画的为啥不会被和谐就sauce和c2d和谐

**Reply by Tsuk1ko** [reply]
设置一下缩略图反和谐 `bot.antiShielding`

**Reply by Tsuk1ko** [reply]
> 但pixiv和链接是红的不知道会不会被封

更新到最新版，很早就已经有处理 pixiv 红链了

**Reply by xiaorunrun520** [reply]
不管怎么防和谐都发不出去，更新时提示
There is no tracking information for the current branch.
Please specify which branch you want to merge with.
See git-pull(1) for details.

    git pull <remote> <branch>

If you wish to set tracking information for this branch you can do so with:

    git branch --set-upstream-to=<remote>/<branch> master

**Reply by Tsuk1ko** [reply]
你不是通过 `git clone` 下载项目的？那要不重新 `git clone` 部署一遍，要不重新下 zip 覆盖一遍
